src/main/java/edu/hm/hafner/analysis/parser/checkstyle/CheckStyleParser.java

```diff
@@ -71,8 +71,8 @@ private Report convert(final CheckStyle collection) {
             if (isValidWarning(file)) {
                 for (Error error : file.getErrors()) {
                     IssueBuilder builder = new IssueBuilder();
-                    mapPriority(error).ifPresent(builder::setSeverity);
 
+                    builder.setSeverity(Severity.guessFromString(error.getSeverity()));
                     String source = error.getSource();
                     builder.setType(getType(source));
                     builder.setCategory(getCategory(source));
```

Comment:
The old method `setSeverity` needs to be removed as well.

Reply:
Sorry: the method that needs to be removed is `mapPriority`

Reply:
Ok sure, already wondered about the duplicated comment.